documentation/modules/upgrading-mtv-ui.adoc

```diff
@@ -69,3 +69,7 @@ forklift-controller-788bdb4c69-mw268  2/2    Running  0         2m
 forklift-operator-6bf45b8d8-qps9v     1/1    Running  0         5m
 forklift-ui-7cdf96d8f6-xnw5n          1/1    Running  0         2m
 ----
++
+. If you added VMware source providers to MTV 2.2, upgrading to MTV 2.3 will make the VMware providers have a Critical state. To fix this problem: Edit the VMwhere provider by adding a VDDK `init` image and verifying the certificate of the VMware provider. For more information see link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.2/html/installing_and_using_the_migration_toolkit_for_virtualization/migrating-vms-web-console#adding-source-provider_vmware[Addding a VMSphere source provider]
```

Comment:
1. Use attributes. Done.
2. Version numbers -- I'd prefer to stick with explicit numbers rather than adding a new attribute for the past version (2.2). If this issue continues version after version, an extra attribute would make more sense.[So no change]
3. Is there a reason to prefer an xref over a link? I thought we don't use xrefs in modules.